### DISCUSSION TITLE
When trying to link repository, message "Page is taking too long to load" with Unicorn

### DISCUSSION THREAD

**Discussion by tiberius-towers** [community/community#35306] [Other Feature Feedback, Questions, & Ideas]
Heya! 

Have a little problem with deploying my repository to Netlify. I'm choosing GitHub on their end, then the pop-up "Authorized" shown from GitHub, but then page turns into an error one with unicorn and "Page is taking too long to load". Happens each time I try to connect repository there, even though all Netlify apps are set up in Github. 

Any tips how to fix this? (collaborator settings are no-go, because my projects has none but me, and it's the only project I have). Thanks. 

![image](https://user-images.githubusercontent.com/81242099/194243138-880fab8f-85c0-4714-bc05-bc0a85b39aa0.png)

**Comment by 3kh0** [comment]
This happens every so often, here are a few quick fixes that sometimes help.

- Reload the page (easiest in the book)
- Reload and clear cache, in chrome do ctrl+shift+r
- Disconnect from wifi, and reconnect
- Open cmd and do `ipconfig/release` and then `ipconfig/renew`
- Turn off your computer, wait 10 seconds, and turn it back on.
- Try a different browser
- Clear cookies and site data from github.com
- Reboot your router

These are some quick and easy fixes, this issue happens from time to time again, and these fixes seem to fix it for me.

*If this helped please mark this as the answer!* ❤️
